### DISCUSSION TITLE
Error when login in login using xampp

### DISCUSSION THREAD

**Discussion by NelsonChad** [filamentphp/filament#12964] [Help] [answered]
### Package

filament/filament

### Package Version

v3.2.0

### Laravel Version

11.8.0

### Livewire Version

v3.0.0

### PHP Version

8.2.12

### Problem description

After all the configuretions of the filament, when when go to 
`http://localhost/filament-admin/public/admin/login`
and click in Login button it shows the error:

Symfony\Component\HttpKernel\Exception\ MethodNotAllowedHttpException
The POST method is not supported for route admin/login. Supported methods: GET, HEAD.

### Expected behavior

Go to admin dashboard 

### Steps to reproduce

- copy the projecto to xampp/htdocs
- start start xampp and localhost/filament-admin/public/admin
- it goes to login page
- enter the user and password
- click login

### Reproduction repository

https://github.com/NelsonChad/filament-admin.git

### Relevant log output

```shell
C:\xampp\htdocs\filament-admin\public\index.php:17

<?php
use Illuminate\Http\Request;

define('LARAVEL_START', microtime(true));

// Determine if the application is in maintenance mode...

if (file_exists($maintenance = __DIR__.'/../storage/framework/maintenance.php')) {

    require $maintenance;

}



// Register the Composer autoloader...

require __DIR__.'/../vendor/autoload.php';

// Bootstrap Laravel and handle the request...

(require_once __DIR__.'/../bootstrap/app.php')

    ->handleRequest(Request::capture());
```

**Comment by leandrocfe** [comment, marked as answer]
Maybe the issue is `/public` in the url. Try to run `php artisan serve` command in the terminal and access `http://localhost:8000/` in the browser

**Reply by NelsonChad** [reply]
Actually that was the problem, thanks for your reply, I was trying to implant it with xampp.

**Reply by Mohitbamania** [reply]
What should be do if i wanted to run it through xampp? I have added full path in virtual host for this project.
